### DISCUSSION TITLE
Huge user/.chace folder (100GB+)

### DISCUSSION THREAD

**Discussion by pcace** [borgbase/vorta#2170] [FAQ]
Hi there, i have Vorta running for the last years, and now realized, that there is a huge (100GB+) .cache/borg folder in my user directory. 
How what can i do to make it go away? can i just delete it?

thanks a lot!

**Comment by goebbe** [comment]
The local .cache/borg directory can be deleted, safely. 
However, next time you create a backup, the backup will take much longer, since the cache will be re-created, which will take some time and is computationally expensive.

Note that in .cache/borg there is exactly one folder per repository. If there are more folders than used repositories, you may have cache files from old repositories. 

If your .cache/borg folder contains a lot of cached files from old/ unused repos, it may be worth to delete/ recreate the .cache/borg folder to get rid of the old cache file and save some space. 

By the way: More than 100GB of cache is huge. For comparison, my 80 GB backup leads to a cache of 65 MB.
